### PR TITLE
add co-chairs

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ The CNF Conformance program will evaluate cloud nativeness based on the set of b
 
 ## Meetings
 
+### Current Co-Chairs
+Service Provider Co-Chair: Jeffrey Saelens
```
Comment (lixuna) [inline README.md]: Do we want to add bullet points?

* Service Provider Co-Chair: Jeffrey Saelens
* CNF Developer Co-Chair: Ian Wells
* Kubernetes Co-Chair: Taylor Carpenter

**Review by jeffsaelens — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ The CNF Conformance program will evaluate cloud nativeness based on the set of b
...
+CNF Developer Co-Chair: Ian Wells
+Kubernetes Co-Chair: Taylor Carpenter
```
Comment (xmulligan) [inline README.md]:
```suggestion
* Service Provider Co-Chair: Jeffrey Saelens
* CNF Developer Co-Chair: Ian Wells
* Kubernetes Co-Chair: Taylor Carpenter
```

File: README.md (hunk 1 above)
Comment (xmulligan) [reply README.md]: yes, good point :)

**Comment by iawells** [comment]
I am not going to object to doing it this way, but I wonder if adding this in the charter makes sense (we're not changing the charter when the co-chairs change), and it would additionally mean we can't be inconsistent if it's only in one place.

**Comment by xmulligan** [comment]
@iawells I agree. I think the Governance should be pulled out of the charter as a separate file similar to the contributors.md. That would also solve your issue

**Comment by iawells** [comment]
If anyone figures out how to use include files in markdown, our lives will be easier (though unfortunately I don't think this is a feature of github's markdown engine)

**Comment by taylor** [comment]
@iawells I agree with governance being pulled out of the charter. Are you okay with addressing that in a new PR?

**Review by fkautz — APPROVED** [review]

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**